### DISCUSSION TITLE
Optimizaed angle of Attack

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#303] [General]
Hi Professor He,
 
I had a question regarding angle of attack variable in DAFoam.  I am running a 2D optimization case with having trailing edge morphing. Optimization is done, and the Cd is minimized, however, the angle of attack changes considerably for example it changes from 3 degrees to  0.313.  Is it normal ? Because when I run a simple simulation in openfoam with AOA= 0.313 without optimization,  the drag is lower than the optimized drag.
 
 
I would appreciate if you help me with this question.
 
Here is the attached opt_IPOPT results and the script below shows the variation of angle of attack which has changed from 3 degees to  0.313 degrees.
[opt_IPOPT.txt](https://github.com/mdolab/dafoam/files/8544094/opt_IPOPT.txt)
![Untitled](https://user-images.githubusercontent.com/97816713/164795298-9e732b57-13e5-4380-88ad-62b4c68017f8.png)

**Comment by friedenhe** [comment]
What are the CD and CL from the DAFoam optimization? And what are the CD and CL from the "simple simulation in openfoam with AOA= 0.313 without optimization"?

**Reply by hosseinnegahban** [reply]
For the optimization case the initial AOA= is 3 degrees and CL=0.44147 (constrained) and optimized  Cd= 0.010466 and the optimized AOA =0.313

For the simple simulation without optimization, AOA= 0.313 and CL= 0.16396     and Cd = 0.009126

**Reply by friedenhe** [reply]
Comparing two designs' Cd is only meaningful if they have the same lift.... If you set AOA=0 degree, you get an even lower drag...

**Reply by hosseinnegahban** [reply]
How about Cl/Cd comparison? When I compare the Cl/Cd of the two designs the optimized one is 42 and the simple one 18. Is this comparison logical?
